README.md

```diff
@@ -1 +1,5 @@
-# gneiss
\ No newline at end of file
+# gneiss
+
+Canonically pronouced *nice*
+
+A compositional statistics and visualization toolbox
```

Comment:
What kind of visualizations are you planning to add? Should they be their own repo or part of another one?